pkgs/misc/tmux-plugins/default.nix

```diff
@@ -197,12 +197,12 @@ in rec {
 
   dracula = mkTmuxPlugin rec {
     pluginName = "dracula";
-    version = "2.3.0";
+    version = "3.0.0";
     src = fetchFromGitHub {
       owner = "dracula";
       repo = "tmux";
       rev = "v${version}";
-      sha256 = "IrNDBRopg9lgN5AfeXbhhh+uXiWQD2bjS1sNOgOJsu4=";
+      sha256 = "VY4PyaQRwTc6LWhPJg4inrQf5K8+bp0+eqRhR7+Iexk=";
```

Comment:
The sri hash should not be crippled and keep it's prefix and we should use the hash argument

Reply:
Will change, thanks.